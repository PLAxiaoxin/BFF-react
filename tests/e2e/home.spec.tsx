// import React from 'react'
// import {render, cleanup} from '@testing-library/react'
// import Home from '@components/Home';
 
//  afterEach(cleanup)
 
//  it('should take a snapshot', () => {
//     const {asFragment} = render(<Home />)
//     expect(asFragment()).toMatchSnapshot()
// });


// describe('Google', () => {
//   beforeAll(async () => {
//     await page.goto('https://google.com');
//   });

//   it('should be titled "Google"', async () => {
//     await expect(page.title()).resolves.toMatch('Google');
//   });
// });
