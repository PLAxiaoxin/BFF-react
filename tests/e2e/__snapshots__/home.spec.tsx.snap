// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="home-container"
  >
    This is Home Page.
    <br />
     
  </div>
</DocumentFragment>
`;
